.github/workflows/ci.yml

```diff
@@ -185,6 +185,8 @@ jobs:
           toolchain: ${{ env.RUST_VERSION }}
           targets: "wasm32-unknown-unknown"
           components: "clippy"
+      - name: Install wasm-tools
+        run: cargo install wasm-tools@1.228.0
```

Comment:
Consider using install-action instead for a bug speed up over cargo install

https://github.com/taiki-e/install-action

Reply:
I considered that but `wasm-tools` isn't supported https://github.com/taiki-e/install-action/blob/main/TOOLS.md. But we could perhaps contribute upstream to add it. Though I would save that for a follow-up.

Reply:
Alright so now we're skipping this entirely at the cost of breaking forward compatibility for the nodes. So we can skip this extra CI bloat.